WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="personal-records">
+  <h3 class="text-center"><%= t 'persons.show.personal_records' %></h3>
+  <div class="table-responsive">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th class="event"><%= t 'competitions.results_table.event' %></th>
```

Comment:
I believe the php code currenty links to the individual's results for this event here.